cmd/kube-spawn/setup.go

```diff
@@ -85,6 +85,29 @@ func doCheckK8sStableRelease(k8srel string) {
 	}
 }
 
+// The kubelet option --fail-swap-on=false is needed for 2 cases:
+//  - "dev" version, which is a self-built K8s cluster
+//  - release "1.8.0" or newer
+func isFailSwapOnFalseNeeded(k8srel string) bool {
+	if utils.IsK8sDev(k8srel) {
+		return true
+	}
+
+	v, err := semver.NewVersion(k8srel)
+	if err != nil {
+		return false
+	}
+
+	k8sFailSwapOnVersion := "1.8.0"
+	c, err := semver.NewConstraint(">=" + k8sFailSwapOnVersion)
```

Comment:
Since this is just a static string, we can ignore the error. Also, we can just write `>= 1.8.0` here.